.gitlab/source_test/linux.yml

```diff
@@ -80,15 +81,24 @@ tests_deb-x64-py3-fast:
   extends:
     - .rtloader_tests
     - .linux_tests
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
-  tags: ["arch:amd64"]
+    - .linux_x64
   needs: ["go_deps", "go_tools_deps"]
   variables:
     PYTHON_RUNTIMES: '3'
     CONDA_ENV: ddpy3
-    EXTRAPARAMS: '--only-modified-packages'
+    EXTRA_OPTS: '--only-modified-packages'
+    TEST_OUTPUT_FILE: test_output_fast.json
   allow_failure: true
 
+stats-fast-tests-deb-x64-py3:
+  stage: source_test_stats
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/datadog-ci-uploader$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
```

Comment:
nit: you could also extend `.linux_x64` here

Reply:
I almost changed it ahah. But then realised that I am not using the same image to be able to use datadog python api